### PR TITLE
(chango) add network config and role

### PR DESCRIPTION
**Pull request by dtapiacl**

### CODE REVIEW

**Review by jhoblitt — APPROVED** [review]

File: hieradata/cluster/chango/role/rke.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+classes:
+  - "profile::core::sysctl::rp_filter"
+profile::core::sysctl::rp_filter::enable: false
+profile::core::rke::enable_dhcp: true
```
Comment (jhoblitt) [inline hieradata/cluster/chango/role/rke.yaml]: Everything above this line isn't needed on chango.

File: hieradata/cluster/chango/role/rke.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+profile::core::rke::version: "1.4.6-rc4"
```
Comment (jhoblitt) [inline hieradata/cluster/chango/role/rke.yaml]: This single line can be moved to `cluster/chango.yaml` and then this file could be removed completely.

File: hieradata/cluster/chango.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+clustershell::groupmembers:
+  chango: {group: "chango", member: "chango[01-03]"}
+cni::plugins::checksum: "f3a841324845ca6bf0d4091b4fc7f97e18a623172158b72fc3fdcdb9d42d2d37"
+cni::plugins::enable: ["macvlan"]
+cni::plugins::version: "1.2.0"
+profile::core::common::manage_powertop: true
+profile::core::ospl::enable_rundir: true
```
Comment (jhoblitt) [inline hieradata/cluster/chango.yaml]: This isn't needed, it is only for running DDS.

File: hieradata/cluster/chango.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+---
+clustershell::groupmembers:
+  chango: {group: "chango", member: "chango[01-03]"}
+cni::plugins::checksum: "f3a841324845ca6bf0d4091b4fc7f97e18a623172158b72fc3fdcdb9d42d2d37"
+cni::plugins::enable: ["macvlan"]
+cni::plugins::version: "1.2.0"
```
Comment (jhoblitt) [inline hieradata/cluster/chango.yaml]: As there are no users of multus on this cluster, all of the `cni::*` keys should be removed.

**Review by jhoblitt — CHANGES_REQUESTED** [review]

File: hieradata/cluster/chango/role/rke.yaml (hunk 1 above)
Comment (dtapiacl) [reply hieradata/cluster/chango/role/rke.yaml]: Cleaned up and updated.

File: hieradata/cluster/chango/role/rke.yaml (hunk 2 above)
Comment (dtapiacl) [reply hieradata/cluster/chango/role/rke.yaml]: `profile::core::rke::version: "1.4.6-rc4"` was moved to cluster/chango.yaml

File: hieradata/cluster/chango.yaml (hunk 3 above)
Comment (dtapiacl) [reply hieradata/cluster/chango.yaml]: Cleaned up.

File: hieradata/cluster/chango.yaml (hunk 4 above)
Comment (dtapiacl) [reply hieradata/cluster/chango.yaml]: Removed from configuraiton.

File: hieradata/cluster/chango.yaml [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+clustershell::groupmembers:
+  chango: {group: "chango", member: "chango[01-03]"}
+profile::core::rke::version: "1.4.6-rc4"
+nm::connections:
```
Comment (jhoblitt) [inline hieradata/cluster/chango.yaml]: In the scenario where there is only a single interface and its using dhcp, no network configuration is needed as NM defaults to being an dhcp client on active links.  The best configuration is usually no-configuration.

**Review by jhoblitt — CHANGES_REQUESTED** [review]

File: hieradata/cluster/chango.yaml (hunk 5 above)
Comment (dtapiacl) [reply hieradata/cluster/chango.yaml]: Network config was removed as advised.

**Review by jhoblitt — APPROVED** [review]

**Pull request merged**